src/Input.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import Input from './Input';
+
+describe('Input', () => {
+  const handleClick = jest.fn();
+  const onChange = jest.fn();
+
+  function renderInput({ value }) {
+    return render((<Input
+      value={value}
+      handleClick={handleClick}
```

Comment:
Input 컴포넌트에서는 onClick으로 받고 있는 것 같아요.

```suggestion
      onClick={handleClick}
```